.github/workflows/breaking-change-review.yml

```
@@ -0,0 +1,90 @@
+name: PR Review Workflow for Breaking Changes
+
+on:
+  pull_request_review:
+    types: [submitted]
```

We probably want all 3 event types:
```suggestion
    types: 
      - submitted
      - edited
      - dismissed
```